tests/test_transform.py

```diff
@@ -0,0 +1,33 @@
+import os
+import subprocess
+import yaml
+from jsonschema.exceptions import ValidationError
+
+from utils import constants
+from jsonschema import validate
+
+
+# Polarion TC 376
+def test_transform_xml_rules_to_yaml():
+    kantra_path = os.getenv(constants.KANTRA_CLI_PATH)
+    custom_rule_path = os.path.join(os.getenv(constants.PROJECT_PATH), 'data/xml', 'javax-package-custom.windup.xml')
+    output_path = os.getenv(constants.REPORT_OUTPUT_PATH)
+
+    command = kantra_path + ' transform rules --input ' + custom_rule_path + ' --output ' + output_path
```

Comment:
@abrugaro I would suggest to use f-strings here (more clear IMO)
Should be something like: `f"{kantra_path} transform rules --input {custom_rule_path} --output {output_path}"`

Reply:
I totally agree with you! thanks! solved in 0b9167e98ca5d223d14c8eefce3d7751a416ffd1